src/emit/polkadot/target.rs

```diff
@@ -223,9 +213,7 @@ impl<'a> TargetRuntime<'a> for PolkadotTarget {
             scratch_len.into()
         );
 
-        log_return_code(binary, "seal_get_storage: ", exists);
-
-        let exists = binary.builder.build_int_compare(
+        let condition = binary.builder.build_int_compare(
```

Comment:
`exists` was supposed to mean "does the key exist". `condition` is not very clear